example_app_frames/src/main/java/com/checkout/example/frames/ui/screen/CVVTokenizationScreen.kt

```diff
@@ -0,0 +1,92 @@
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavHostController
+import com.checkout.base.model.CardScheme
+import com.checkout.base.model.Environment
+import com.checkout.example.frames.ui.component.ClickableImage
+import com.checkout.example.frames.ui.component.TextComponent
+import com.checkout.example.frames.ui.theme.DarkBlue
+import com.checkout.example.frames.ui.theme.FramesTheme
+import com.checkout.example.frames.ui.utils.PUBLIC_KEY
+import com.checkout.frames.R
+import com.checkout.frames.cvvcomponent.CVVComponentApiFactory
+import com.checkout.frames.cvvcomponent.models.CVVComponentConfig
+
+@Suppress("MagicNumber", "LongMethod")
+@Composable
+fun CVVTokenizationScreen(navController: NavHostController) {
+    val context = LocalContext.current
+    FramesTheme {
+        // A surface container using the 'background' color from the theme
+        Surface(
+            modifier = Modifier.fillMaxSize(), color = MaterialTheme.colorScheme.background
+        ) {
+            Column(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .verticalScroll(state = rememberScrollState()),
+                verticalArrangement = Arrangement.Top,
+                horizontalAlignment = Start
+            ) {
+
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(start = 10.dp, end = 24.dp, bottom = 14.dp),
+                    horizontalArrangement = Arrangement.Start
+                ) {
+
+                    ClickableImage(
+                        painter = painterResource(id = R.drawable.ic_back_arrow),
+                        contentDescription = "My Image",
+                        onClick = { navController.popBackStack() },
+                        paddingValues = PaddingValues(top = 20.dp, bottom = 14.dp),
+                    )
+
+                    TextComponent(
+                        titleResourceId = R.string.cvv_component,
+                        fontSize = 20,
+                        fontWeight = FontWeight.SemiBold,
+                        paddingValues = PaddingValues(top = 20.dp, start = 10.dp, bottom = 14.dp),
+                        textColor = DarkBlue
+                    )
+                }
+
+                Spacer(Modifier.height(20.dp))
+
+                val cvvComponentApi = CVVComponentApiFactory.create(PUBLIC_KEY, Environment.SANDBOX, context)
+
+                val cvvComponentConfig = CVVComponentConfig(
+                    cardScheme = CardScheme.fetchCardScheme(cardSchemeValue = "Visa"),
+                    onCVVValueChange = { isValidCVV ->
+                        println(isValidCVV)
+                    }
+                )
+
+                val mediator = cvvComponentApi.createComponentMediator(cvvComponentConfig)
```

Comment:
shall we avoid declaring non-composable variables in a composable function?

Reply:
Yes sure.